Services/CO.CDP.OrganisationInformation.Persistence/Organisation.cs

```diff
@@ -46,4 +50,67 @@ public record OrganisationContactPoint
         public string? Telephone;
         public string? Url;
     }
+
+    [Owned]
+    public record SupplierInformation : IEntityDate
+    {
+        public required SupplierType SupplierType { get; set; }
+        public List<OperationType> OperationTypes { get; set; } = [];
+        public bool? CompletedRegAddress { get; set; }
```

Comment:
Instead of making all these bools nullable, I suggest we make them non-null but set defaults to false.

Anytime we can provide a reasonable non-nullable default, we should use it.

Reply:
We need nullable bool for first time else it will be always No answer.

Reply:
It's `false` the first time. We're not storing an answer here, just the fact that the answer was given. 

For example, 

`CompletedVat` == `false` means the supplier has not answered the question yet.
`CompletedVat` == `true` means they answered the question. The existence of vat identifier tells us how they answered.

For these boolean flags, there's only two states (answer given / answer not given). By using `null` as one of the states, we will only have either `null` or `true`. Imo, it's better to use `false` and `true` 🤷

Reply:
Done